components/server/src/auth/login-completion-handler.ts

```diff
@@ -88,7 +88,7 @@ export class LoginCompletionHandler {
         if (authHost) {
             increaseLoginCounter("succeeded", authHost);
 
-            /** no await */ trackLogin(user, request, authHost, this.analytics).catch((err) =>
+            /** no await */ trackLogin(user, request, authHost, this.analytics, this.subscriptionService).catch((err) =>
                 log.error({ userId: user.id }, err),
```

Comment:
[nit] Does it make sense to add more human context to the logged error? Here we might just get the user ID and the raw error, but we could say "Failed to track login: <error>"

Reply:
hi @easyCZ, thanks for taking the time to leave these comments! great point, i added a little bit of context.